internal/handlers/common/error.go

```diff
@@ -108,11 +108,19 @@ func (e *Error) Document() *types.Document {
 	return d
 }
 
+// ProtoErr represents protocol error type.
+type ProtoErr interface {
```

Comment:
Why do we define a common interface between two implementations?

Reply:
That was a question about ProtoErr interface definition in the source ode file – it was sandwiched between Error and WriteErrors definitions. That's not the case now